### ISSUE TITLE
First release TODO

### ISSUE THREAD

**Issue by knuxify** [aspinwall-ui/lapel#1]
This issue outlines the tasks that need to be done before the app is ready for a proper release.

  - [X] Assistant
    - [X] Show message time (74df85910f3430a0da41ef05f2bdfb9988973d08)
    - [X] Fix going into voice control mode (7374b66087aaac30aec93996dc699b1dd553f619)
    - [X] Add voice control GUI (11af56066c298f2f720f335a36ee11161072b19e)
    - [X] Send button in entry (cd56d283c17e3679c13a69cdb151406b622db8a0)
    - [X] Auto-scroll to the bottom (ada8e3c38c71089b85e34b1a313d05b3f652bd90, 591b1d3 - still kinda janky, will need fixes)
    - [X] Stop recording when message is being typed/sent - should prevent issues with e.g. confirmation dialogs (743a698731229a32ec5a1dd490bbc95175fe2eaf)
  - [X] Skills menu (9a77227d32db26b0cfddd8acd9c73a36410841ee, 91d24bb581da97a6d6b454a15e69f9c517c13fe7, more improvements yet to come)
     - [X] Add skill icon display (26274a9969853389d5fbfdf764ac5304e986b18a)
     - [X] Search (4a86978035506176c8e8c68c30765b5da0434c68)
  - [X] Preferences menu (21fb7cb2dfc85b5605c39ce593aa0e998f8902ad)
    - [X] Websocket address selection (21fb7cb2dfc85b5605c39ce593aa0e998f8902ad)
  - [X] About menu
  - [X] Wrappers
    - [X] Image attachment dialog wrapper (e103c450ec514becf2ec648551b08734cfcb7047)
    - [X] Better handling for hiding suggestion wrappers (76c429b752336a1d9c6f0bbb699cc7f20d846808)
  - [X] Background daemon that allows the UI to show up when Mycroft is called, even outside of the app (84d51e0070cf02f5fd37d6cab6bbb9e06ee37b50, 90a72db6b78adff82879e2464d817ce1293e6093, 5ddd8eeef4002c44ea3fe7141248892db2772fd5)
  - [X] Pairing dialog (ce63001b43d66872366680505bc646f15485e56d)
  - [X] Error handling (9475b7e7a7d1eb90f3dec712d31b3cf429d8430c)
    - [X] Make it appear in all views, not just the assistant (610485ff92f8dfbd51bd4848fb45ab97379a1deb)
    - [X] Lock view switcher buttons (4806696e60b2875d04d45e03775165091ec6c90d)
    - [X] Add custom checker for Mycroft presence (193091b7a1bf38514d1d9552eaf6ca7f9b03c5fe)
    - [X] Re-create client when the connection is dropped (0ac7c60f1e6b044d87d13f23a69f662316867bf6)
  - [X] Add commandline argument to start as daemon (ef2474a64e94c53d86ebcb016a78f4fda2c40584)

## Postponed for next release
* TTS/SST settings? (Plasma Mobile seems to have something related, according to mycroft-gui commits - what is it, and does GNOME offer something similar?)
* Text parsing workarounds (many skills, such as showing the IP or installed version, change the text to make the numbers more readable to text-to-speech - we need to turn it back to a more readable form)
* Proper Flatpak support

**Comment by knuxify** [comment]
New release is planned for 2022-08-07, giving translators a bit of extra time to catch up with missing strings.
